sagenb/notebook/notebook_object.py

```diff
@@ -12,11 +12,15 @@
 #                  http://www.gnu.org/licenses/
 #############################################################################
 
-import time, os, shutil, signal, tempfile
+import time
+import os
+import shutil
+import signal
+import tempfile
 
-import notebook as _notebook
+from . import notebook as _notebook
 
-import run_notebook
+from . import run_notebook
```

Comment:
This is relative import syntax, but the PR said something about "absolute import".  Did you mean relative import?